frontends/web/src/components/guide/entry.tsx

```diff
@@ -55,7 +56,14 @@ export const Entry: FunctionComponent<TProps> = props => {
           <div className="flex-1">
             {entry.text.trim().split('\n').map(p => <p key={p}>{p}</p>)}
             {entry.link && (
-              <p><A data-testid="link" href={entry.link.url}>{entry.link.text}</A></p>
+              <p>
+                <A
+                  className={style.link}
+                  data-testid="link"
+                  href={entry.link.url}>
+                  {entry.link.text}
+                </A>
+              </p>
```

Comment:
I haven't tested, do we need `p` here?

Reply:
The only reason is that the font size is a bit smaller in guide answers, without `<p>` element the link text is a bit bigger than the normal text. I thought it should keep inheriting font-size here and no need to add custom class only to change font size.